src/user-guide/interventions/index.rst

```diff
@@ -0,0 +1,433 @@
+       initial cycle point = 1
+       runahead limit = P1
+       [[graph]]
+           P1 = """
+               start => switch
+               switch:normal? => normal
+               switch:alternate? => alternate
+               normal | alternate => end
+   
+               end[-P1] => start
+           """
+   
+   [runtime]
+       [[start]]
+           script = """
+               if [[ $CYLC_TASK_CYCLE_POINT -eq 1 ]]; then
+                   sleep 3
+               fi
+           """
+       [[switch]]
+           script = cylc message -- normal
+           [[[outputs]]]
+               normal = normal
+               alternate = alternate
+       [[normal, alternate]]
+       [[end]]
+
+:Example:
+   I have a :term:`branched workflow <graph branching>`, I want to direct it to
+   take a particular path ahead of time:
+
+   .. code-block:: cylc-graph
+   
+      start => switch
+   
+      # by default, the workflow follows the "normal" path
+      switch:normal? => normal
+   
+      # but it can follow an alternate route if desired
+      switch:alternate? => alternate
+   
+      normal | alternate => end
+
+:Solution:
+   Set "succeeded" as well as the desired output(s) on the task and use the
+   ``-wait`` option. Cylc will follow the desired path when it gets there.
+
+   .. note::
+   
+      We set the "succeeded" output to prevent the task from being re-run when the
+      flow approaches it.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: set-a-switch-task.gui.gif
+         :width: 75%
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc set --wait --output=succeeded,alternate <workflow>
+
+
+.. _interventions.edit-a-tasks-configuration:
+
+Edit a Task's Configuration and Re-Run It
+-----------------------------------------
+
+:Example:
+   I'm developing a task and need to be able to quickly make changes and re-run
+   the task.
+
+:Solution:
+   Edit the task's definition and trigger it.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: edit-a-tasks-configuration.gui.gif
+         :width: 75%
+
+      .. note::
+
+         Any changes you make, apply only to this one instance of the task, not
+         to any future instances.
+
+         To change future instances, either use "broadcast" or see
+         :ref:`interventions.edit-the-workflow-configuration`.
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc broadcast <workflow> -p <cycle> -n <task> -s 'script=true'
+         $ cylc trigger <workflow>//<cycle>/<task>
+
+.. the "|" character adds some vertical whitespace
+
+|
+
+.. _interventions.edit-the-workflow-configuration:
+
+Edit The Workflow Configuration
+-------------------------------
+
+:Example:
+   I want to change the configuration of multiple tasks or add/remove tasks
+   without stopping the workflow or starting it from the beginning again.
```

Comment:
This mentions adding or removing tasks from the graph, which has some caveats, so I think we should link here to the "Restarting or reloading after graph changes" section.

Reply:
Added a note.